web_src/js/features/repo-release.ts

```diff
@@ -21,24 +23,66 @@ function initTagNameEditor() {
   const el = document.querySelector('#tag-name-editor');
   if (!el) return;
 
+  const tagWarning = document.querySelector('#tag-warning');
+  const tagWarningDetailLinks = Array.from(document.querySelectorAll('.tag-warning-detail'));
```

Comment:
```suggestion
  const tagWarningDetailLinks = document.querySelectorAll<HTMLAnchorElement>('.tag-warning-detail');
```

This avoids the `as` further below and array convertion is unnecessary as `for-of` can iterate `NodeList` just fine.